rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/TestSchemaNameCustomizable.java

```diff
@@ -44,7 +43,7 @@ void resetDb() throws SQLException {
   @Test
   void checkCustomSchemaNameIsDefined_Postgres() throws SQLException {
     resetDb();
-    Assume.assumeTrue("Test only executed with Postgres database", isPostgres);
```

Comment:
This is not an assertions. This is an assumption.
Please use `Assumptions.assumeThat(isPostgres).isTrue();` (with a static import)